api/app/controllers/webhook.controller.ts

```diff
@@ -99,18 +104,26 @@ exports.create = async (req, res) => {
       const associatedTranscript = await Transcript.findByPk(
         existingReview.transcriptId
       );
-      associatedTranscript.archivedAt = currentTime;
-      await associatedTranscript.save();
 
-      const creditAmount = await calculateCreditAmount(associatedTranscript);
-      await createCreditTransaction(existingReview, creditAmount);
+      if (associatedTranscript) {
+        associatedTranscript.archivedAt = currentTime;
+        await associatedTranscript?.save();
 
-      return res.sendStatus(200);
+        const creditAmount = await calculateCreditAmount(associatedTranscript);
+        await createCreditTransaction(existingReview, creditAmount);
+
+        return res.sendStatus(200);
+      } else {
+        // Could not find associated transcript
+        res.sendStatus(500);
```

Comment:
We should throw an error here with a message instead of sending a response. The catch blocks already sends a response if there is an error.